Project.toml

```diff
@@ -19,5 +19,5 @@ CreateParametersExt = "ClimaParams"
 ClimaParams = "0.10"
 DocStringExtensions = "0.8, 0.9"
 RootSolvers = "0.4"
-Thermodynamics = "0.12.8"
+Thermodynamics = "0.12.15"
```

Comment:
Why do we need 0.12.15 now?

Reply:
`ClimaAtmos` is pinned to `0.12.14`; I can pin it to this version instead - there have been some recent updates to Thermodynamics which we may wish to test soon.